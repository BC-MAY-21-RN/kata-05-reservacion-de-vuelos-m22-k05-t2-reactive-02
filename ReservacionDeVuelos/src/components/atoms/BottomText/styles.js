import {StyleSheet, Dimensions} from 'react-native';
import colors from '../../../consts/colors';

const styles = StyleSheet.create({
  container: {
    ...{width: '100%', height: 30, marginTop: '2.46%'},
    ...(Dimensions.get('screen').height <= 640
      ? {marginTop: '2.46%'}
      : {marginTop: 35}),
  },
  text1: {
    fontSize: 16,
    color: colors.termscolor,
    marginRight: 3,
  },
  text2: {color: colors.bluetitle, fontSize: 16},
  rowstyle: {flex: 1, flexDirection: 'row', alignSelf: 'center'},
});

export default styles;
