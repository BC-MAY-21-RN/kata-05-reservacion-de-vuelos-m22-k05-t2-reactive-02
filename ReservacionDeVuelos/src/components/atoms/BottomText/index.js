import React from 'react';
import {View, Text, TouchableOpacity} from 'react-native';
import texts from '../../../consts/text';
import styles from './styles';

export default function BottomText({navigation, textComponent, nameScreen}) {
  return (
    <View style={styles.container}>
      <View style={styles.rowstyle}>
        <Text style={styles.text1}>{texts.register.bottomtext}</Text>
        <TouchableOpacity onPress={() => navigation.navigate(nameScreen)}>
          <Text style={styles.text2}>{textComponent}</Text>
        </TouchableOpacity>
      </View>
    </View>
  );
}
