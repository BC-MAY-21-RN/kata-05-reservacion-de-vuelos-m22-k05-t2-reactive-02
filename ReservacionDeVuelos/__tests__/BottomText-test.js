import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BottomText from '../src/components/atoms/BottomText';

describe('Tests BottomText component', () => {
  it('BottomText snapshot', () => {
    const renderBottomText = renderer
      .create(<BottomText nameScreen={''} textComponent={''} />)
      .toJSON();
    expect(renderBottomText).toMatchSnapshot();
  });
});
