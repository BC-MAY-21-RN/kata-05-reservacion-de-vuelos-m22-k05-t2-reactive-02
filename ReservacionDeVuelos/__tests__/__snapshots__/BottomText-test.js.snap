// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests BottomText component BottomText snapshot 1`] = `
<View
  style={
    Object {
      "height": 30,
      "marginTop": 35,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "center",
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#9CA5B4",
          "fontSize": 16,
          "marginRight": 3,
        }
      }
    >
      Already have an account?
    </Text>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#5C6DF8",
            "fontSize": 16,
          }
        }
      />
    </View>
  </View>
</View>
`;
